cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -863,7 +863,7 @@ Slice::CsGenerator::writeUnmarshalCode(Output &out,
     StructPtr st = StructPtr::dynamicCast(type);
     SequencePtr seq = SequencePtr::dynamicCast(type);
```

Comment:
Unrelated to your PR, but it doesn't look like we actually need this `seq` variable here.

Reply:
you are right, I will remove it.